proxy/xml_validator.rb

```diff
@@ -0,0 +1,20 @@
+require 'nokogiri'
+
+module XMLValidator
+
+    def self.validate(xml)
+        xsd = Nokogiri::XML::Schema(File.read("post_joke.xsd"))
```

Comment:
Hardcoded values are typically a bad thing.
A more extensible approach is to pass filename as a parameter to the method.